website/client/src/components/tasks/spells.vue

```diff
@@ -127,7 +130,7 @@
 
       span {
         display: inline-block;
-        width: 50%;
```

Comment:
@JalanshMunshi looks like this change broke the layout:

<img width="918" alt="Schermata 2020-08-09 alle 12 28 43" src="https://user-images.githubusercontent.com/589911/89730110-26ad3580-da3c-11ea-9341-3a7f4946774c.png">

^ this is how it looks in develop

<img width="919" alt="Schermata 2020-08-09 alle 12 31 31" src="https://user-images.githubusercontent.com/589911/89730128-45abc780-da3c-11ea-90c8-3d7fc2639f51.png">

^ this without the `width: 50%;`

Reply:
actually sorry looks like this was committed by @SabreCat